graphs/DeltaArrow/index.js

```diff
@@ -38,14 +41,17 @@ module.exports = React.createClass({
   },
 
   render: function() {
-    var props   = this.props,
-        delta   = props.delta,
-        loading = props.loading,
+    var props      = this.props,
+        delta      = props.delta,
+        loading    = props.loading,
+        emptyState = props.emptyState,
         className,
         text;
 
-    if (loading == true ) {
+    if (loading == true) {
       className = "hui-DeltaArrow--loading";
+    } else if (emptyState == true) {
+      className = "hui-DeltaArrow--emptyState";
     } else if (delta == 0) {
```

Comment:
We should mix logic that is build classnames and controlling render. Also avoid long if else statements. It smells.